opentelemetry-sdk/src/opentelemetry/sdk/trace/export/in_memory_span_exporter.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import threading
+import typing
+
+from .. import Span
+from . import Exporter, ExportResult
+
+
+class InMemorySpanExporter(Exporter):
+    """Implementation of :class:`.Exporter` that saves spans in memory.
+
+    This class can be used for testing purposes. It saves the exported spans
+    in a list in memory that can be retrieve using the
+    :func:`get_finished_spans` method
```

Comment:
```suggestion
    :func:`.get_finished_spans` method.
```

Not entirely sure about the dot inside `:func:` but I think this makes sphinx look in this class first instead of the module.